.azure/docs-linux.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           verbose: true
 
       - task: PublishBuildArtifacts@1
-        displayName: 'Publish docs'
+        displayName: 'Upload docs artifact'
```

Comment:
This change is meant to be less ambiguous because publish could mean pushing to the COS.